testing-guidance/workflow-tests/physics/physics-lkg-workflows.md

```diff
@@ -1,200 +1,466 @@
-| PhysX Materials can be applied to colliders and impact collisions  | <ol><li>Create a PhysX Collider and PhsyX Shape Collider entity setup with appropriate Collider componenets for each PhysX Material under test.</li><li>Add the desired PhysX Material(s) under test to the entities created.</li><li>Set up the scene so that the entities with PhysX Materials under test will collide with something.</li><li>Run the Simulation in Editor Game Mode or in the Game Launcher.</li></ol>                                                     | <ul><li>PhysX Colliders can have PhysX Materials applied.</li><li>PhysX Material properties are applied to Colliders and impact collisions.</li>                                                                                                                                                                   |
----
+| Workflow | Requests | Things to Watch For | Lionbridge Notes |
+| --- | --- | --- | --- |
+| Workflow | Requests | Things to Watch For | Lionbridge Notes |
+| --- | --- | --- | --- |
+| PhysX Colliders collide with expected entities | *   Create an entity for each of the[PhysX Collider shape(s)](https://github.com/o3de/sig-simulation/blob/main/testing-guidance/workflow-tests/physics/physics-lkg-workflows.md#physx-collider-shapes)or[PhysX Shape Collider Shape(s)](https://github.com/o3de/sig-simulation/blob/main/testing-guidance/workflow-tests/physics/physics-lkg-workflows.md#PhysX-Shape-Collider-shapes)under test. <br>*   Add a PhysX Collider or PhysX Shape Collider to the entity for and add their partner[Shape](https://github.com/o3de/sig-simulation/blob/main/testing-guidance/workflow-tests/physics/physics-lkg-workflows.md#PhysX-Shape-Collider-shapes)<br>*   Set up a scene in which entities with colliders will collider onto each other.<br>*   Run the Simulation in Editor Game Mode and in the Game Launcher. | *   PhysX Colliders components are successfully added to entities.<br>*   PhysX Shape Colliders alert users to the need for corresponding shape meshes.<br>*   PhysX Collider with PhysicsAsset shape works when a mesh asset is applied.<br>*   Colliders collide as expected when colliding with entities. |     |
```

Comment:
MD Tables require HTML ordered/unordered list formatting to render correctly.  Please restore the HTML  formatting to get the correct rendering back.

Please see this file when rendered for reference: https://github.com/o3de/sig-simulation/blob/18123396d779dadadb4014b82325e79ee732f9cf/testing-guidance/workflow-tests/physics/physics-lkg-workflows.md